metadata.json

```diff
@@ -46,14 +46,20 @@
       "operatingsystem": "FreeBSD"
     }
   ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.7.1 < 6.0.0"
+    }
+  ],
   "dependencies": [
     {
-      "name": "puppetlabs/concat",
-      "version_requirement": ">= 1.0.1 < 5.0.0"
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.13.1 < 5.0.0"
     },
     {
-      "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 1.0.0 < 5.0.0"
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 4.0.0 < 5.0.0"
```

Comment:
And reason this wouldn't work with an older concat? In general it's nice to have the option to use an older version in case other modules aren't allowing ```4.0.0``` yet.

Reply:
@ekohl It probably still would -- however, once we stop testing against it, may be hard to assert that it still works with it. In general, we tend to drop support for older versions of stuff in modules, but I'm happy to set it to whatever makes sense.
nginx is using `">= 3.0.0 <5.0.0"`

Reply:
@bastelfreak do you want me to back it down to 3.0?

Reply:
3.0 seems to be fine. 4.0 just added datatypes, but has the same functionality